packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,15 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "white" !default;
+$theme-in-page-nav-background-padding: 2 !default;
+$theme-in-page-nav-background-radius: "lg" !default;
+$theme-in-page-nav-bar-color: "secondary-dark" !default;
```

Comment:
I like that this is a customizable setting. However, I am wondering if the default value for this should be in the `primary` family. My thinking is that our component defaults use primary or base colors, and to use secondary here feels like an intentional deviation. 
If this choice to use secondary is meant to communicate that users should *not* be using primary or base here, then we should make sure to note that in our guidance.

Reply:
I think this should be `default` where the default is the text color as calculated by settings

Reply:
Changed to a `default` value that defaults to text color